.github/workflows/rav1e.yml

```
@@ -113,13 +119,17 @@ jobs:
       env:
         LINK: http://www.deb-multimedia.org/pool/main/d/dav1d-dmo
         DAV1D_VERSION: 0.5.1-dmo1
-        DAV1D_DEV_SHA256: feb8fd535ae7747963d3d17ed394dc5bdb3e6163fdb77df787ec72a8ca9aac2e
-        DAV1D_LIB_SHA256: 682fd52fcfd73c225f9aaee200cbe69eceefdc687b8ce03f354731f5288a28ce
+        DAV1D_DEV_SHA256: >-
+          feb8fd535ae7747963d3d17ed394dc5bdb3e6163fdb77df787ec72a8ca9aac2e
+        DAV1D_LIB_SHA256: >-
+          682fd52fcfd73c225f9aaee200cbe69eceefdc687b8ce03f354731f5288a28ce
       run: |
         curl -O "$LINK/libdav1d-dev_${DAV1D_VERSION}_amd64.deb" \
              -O "$LINK/libdav1d3_${DAV1D_VERSION}_amd64.deb"
-        echo "$DAV1D_DEV_SHA256  libdav1d-dev_${DAV1D_VERSION}_amd64.deb" | sha256sum --check
-        echo "$DAV1D_LIB_SHA256  libdav1d3_${DAV1D_VERSION}_amd64.deb" | sha256sum --check
+        echo "$DAV1D_DEV_SHA256  libdav1d-dev_${DAV1D_VERSION}_amd64.deb" |
+        sha256sum --check
+        echo "$DAV1D_LIB_SHA256  libdav1d3_${DAV1D_VERSION}_amd64.deb" |
+        sha256sum --check
         sudo dpkg -i "libdav1d-dev_${DAV1D_VERSION}_amd64.deb" \
                      "libdav1d3_${DAV1D_VERSION}_amd64.deb"
     - name: Install stable
```

This step name is a little misleading for the grcov task.
```suggestion
    - name: Install ${{ matrix.toolchain }}
```